app/views/posts/_task.html.erb

```diff
@@ -0,0 +1,71 @@
+<div class="row">
```

Comment:
please rename to properly state what the file is about.
It's not single task partial, rather it seems like a list of tasks